Modifying the outreach section of the website (#1005)
root-project/web: get_started/index.md

On behalf of the developers, contributors and user community: **welcome to ROOT!**

> If you have never used ROOT before and don't know where to start, we recommend that you first explore: https://github.com/root-project/student-course

Reviewer: ```suggestion
If you have never used ROOT before and don't know where to start, we recommend that you first explore [the ROOT introductory course](https://github.com/root-project/student-course).
```

> For that purpose you don't need to install ROOT on your machine, you can directly run all the examples and exercises 

Reviewer: ```suggestion
For this course, you don't need to install ROOT on your machine. You can directly run all the examples and exercises 
```

> on [SWAN](https://swan.cern.ch){:target="_blank"} (if you have a CERN computing account), otherwise either using github codespaces or binder. 

Reviewer: ```suggestion
on [SWAN](https://swan.cern.ch){:target="_blank"} (if you have a CERN computing account), or otherwise using GitHub Codespaces or Binder. 
```

> The course is written in python. We use jupyter notebooks as the basis to both explain the fundamental concepts and show the code examples. 
> We also provide a few exercises that you can attempt on your own. If you want to know a bit more and complete a few more exercises, 
> we added some extra material that can be found in the above repository as well. 

Reviewer: ```suggestion
The course is written in Python. We use Jupyter Notebooks as the basis to both explain the fundamental concepts and show the code examples. 
We also provide a few exercises that you can attempt on your own. If you want to know a bit more and complete a few more exercises, 
we added some extra material that can be found in the above repository as well. 
```

Once you are a bit more familiar with what ROOT offers and how to use it online, you can take a look at how to install it on your machine 
[install]({{ '/install' | relative_url }}). 

Finally, if you wish to delve a bit further into ROOT functionalities, check the following: 
* The [ROOT beginners' guide (aka "Primer")]({{ '/primer' | relative_url }})
* The [Manual]({{ '/manual' | relative_url }}) where more in depth information can be found
* The full [API Documentation](https://root.cern/doc/master/){:target="_blank"}
* The [tutorials]({{ '/tutorials' | relative_url }})

> Lastly, in case you have a problem or a question, don't hesitate to use the [ROOT Forum](https://root-forum.cern.ch){:target="_blank"}

Reviewer: ```suggestion
Lastly, in case you have a problem or a question, don't hesitate to use the [ROOT Forum](https://root-forum.cern.ch){:target="_blank"}.
```
